src/grbrun.in

```diff
@@ -60,11 +60,6 @@ then
 fi
 
 RUNNER=${BACKENDRUNNER[${BACKENDID}]}
+LD_PATH="@ALP_UTILS_INSTALL_DIR@:${BACKENDRUNENV[${BACKENDID}]}"
 
-EXPENV=${BACKENDRUNENV[${BACKENDID}]}
-if [ -n "${EXPENV}" ]; then
-	export ${EXPENV}
-fi
```

Comment:
I think what is moved here to be part of LD_PATH may not capture all possible uses of EXPENV -- it looks like EXPENV could export something other than LD_LIBRARY_PATH, is it not? If yes, is it not better to retain?

Reply:
`LD_PATH` contains only paths to be prefixed to `LD_LIBRARY_PATH`. To confirm this, I am showing the content of a generated `grbrun` (with anonimized paths and ellipsis for unrelated code parts)

```bash
#!/bin/bash

...

BACKENDS=(
"reference"
"reference_omp"
"hyperdags"
"nonblocking"
"bsp1d"
"hybrid"
)
BACKENDRUNENV=(
"<install path>/install/lib/sequential"
"<install path>/install/lib/sequential"
"<install path>/install/lib/hyperdags"
"<install path>/install/lib/sequential"
"<install path>/install/lib/spmd"
"<install path>/install/lib/hybrid"
)
BACKENDRUNNER=(
""
""
""
""
"<LPF path>/bin/lpfrun -engine mpimsg"
"<LPF path>/bin/lpfrun -engine mpimsg"
)

declare -a ARGS
SHOW="eval"
BACKEND=${BACKENDS[0]}

while [[ $# -gt 0 ]]; do
    ...
done

BACKENDID=-1
for i in "${!BACKENDS[@]}"; do
    ...
done

if [[ ${BACKENDID} -eq -1 ]]
    ...
fi

RUNNER=${BACKENDRUNNER[${BACKENDID}]}
LD_PATH="<install path>/install/lib:${BACKENDRUNENV[${BACKENDID}]}"

${SHOW} LD_LIBRARY_PATH="${LD_PATH}:${LD_LIBRARY_PATH}" ${RUNNER} "${ARGS[@]}" "$@"
```

Reply:
OK for me -- as discussed, any other envvars can be introduced separately via their own MRs. Clicking resolve here=)